com.github.jkotra.unlockr.yml

```diff
@@ -0,0 +1,29 @@
+app-id: com.github.jkotra.unlockr
+runtime : org.gnome.Platform
+runtime-version : '42'
+sdk : org.gnome.Sdk
+command: unlockr
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --filesystem=home:ro
```

Comment:
since the app uses Gtk4, it support portals. It shouldn't even need any permissions.

Reply:
The home one for sure - the means of operation does need to get permissions on a writable location given that it decrypts and saves the PDF and doesn't want to overwrite the existing encrypted file.

Reply:
Yes, i removed home but kept `xdg-documents`. PDF gets automatically saved to Documents (overwrite if it already exists)

This is the behaviour for now, might change in future.